docs/redis.md

```diff
@@ -24,6 +24,14 @@ When this happens, Heroku begins sending deprecation notices via email. Steps to
 They have been adapted from @tsibley's [notes on the 5 → 6 upgrade](https://github.com/tsibley/blab-standup/blob/17eb1690b70ca25aa7be7526b7e140e43cf0a1e6/2023-02-17.md),
 which is based on the [`--fork` upgrade method](https://devcenter.heroku.com/articles/heroku-redis-version-upgrade#upgrade-using-a-fork) described in Heroku's own documentation.
 
+> [!NOTE]
+> Heroku provides an [in-place upgrade
+> method](https://devcenter.heroku.com/articles/heroku-redis-version-upgrade#upgrade-using-redis-upgrade)
+> that is much simpler than the steps below. However, there is no option to roll
+> back in case anything unexpected happens. If we find ourselves going through
+> this process more often without any failures and this becomes too tedious, it
+> may be worth switching to use the in-place upgrade method.
```

Comment:
Linking back to our discussion of the tradeoffs here (https://github.com/nextstrain/private/issues/121#issuecomment-2330682764) will probably be helpful in the future.

Reply:
Done: https://github.com/nextstrain/nextstrain.org/blob/57f9482601246f733071632949efe39ad56ae94a/docs/redis.md?plain=1#L33-L37